internal/config/options.go

```diff
@@ -171,11 +174,23 @@ var defaultOptions = Options{
 	GRPCClientDNSRoundRobin: true,
 }
 
+func NewEmptyOptions() *Options {
```

Comment:
I actually think this should be what `NewOptions` does and the functionality of `NewOptions` could be done with something like `NewMinimalOptions`.  Thoughts?

Reply:
Agree!